### PR TITLE
Load fonts from url getted with `get_url()` first.

### LINKED ISSUE #34: It request fonts with invalid path.
It seems to be a problem when uri has path.

e.g., when uri is `example.com/blog`, it *should* get fonts from `example.com/blog/assets/fonts/some-font.woff`.
but it sends request to `example.com/assets/fonts/some-font.woff` and receives `HTTP 404`.

You may want to visit [my page](https://p-inks.github.io/pages/) for details\. (that hasn't any contents)

### PR DESCRIPTION
**Pull request by p-inks**
This makes website loads font with correct url. (Fixes #34)
To improve a little of speed, I have tried to load the font first.

### CODE REVIEW

**Comment by ejmg** [comment]
alright, as it stands, i can't really capture this problem with local testing. furthermore, as [i've commented on originating issue](https://github.com/ejmg/zerm/issues/34#issuecomment-896175901), i don't understand why the issue was occurring to begin with.

**however**, the changes you introduce deploy just fine both locally for me and on the vercel deployment. furthermore, adding `get_url` makes zola explicit in where it gets the fonts assets. finally, while making `crossorigin="anonymous"` explicit [shouldn't be necessary](https://github.com/ejmg/zerm/issues/34#issuecomment-894923790), for some reason it solved the issue of double loading fonts in your situation. 

in short, nothing of substance really changed with this PR but the potential problem of double loading fonts was resolved. as far as i care, merging this is a non-issue and helps with a potentially weird edge case while making some behaviors of Zola/crossorigin explicit.

thank you for the PR!
